emex_upload/services.py

```
@@ -0,0 +1,658 @@
+
+                if task:
+                    processed_rows = i + len(chunk)
+                    progress = int((processed_rows / total_rows) * 100)
+                    task.update_state(
+                        state='PROGRESS',
+                        meta={'step': 'INSERTING', 'status': 'IN_PROGRESS', 'progress': progress}
+                    )
+            
+            return True, f"Успешно загружено {total_rows} строк."
+
+    except Exception as e:
```

Ruff (0.13.3)

Do not catch blind exception: `Exception`

(BLE001)